apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notice.js

```diff
@@ -73,11 +51,7 @@ function GlobalStylesNoticeComponent() {
 						<ExternalLink
 							href={ wpcomGlobalStyles.upgradeUrl }
 							target="_blank"
-							onClick={ () =>
-								recordTracksEvent( 'calypso_global_styles_gating_notice_upgrade_click', {
-									context: 'site-editor',
-								} )
-							}
+							onClick={ () => recordUpgradePreSaveNoticeClick() }
```

Comment:
Nitpick (non-blocking): The arrow function is not needed, so we could simplify this a little bit.
```suggestion
							onClick={ recordUpgradePreSaveNoticeClick }
```